src/codegen.cpp

```diff
@@ -5481,7 +5478,7 @@ static std::unique_ptr<Module> emit_function(
     DebugLoc noDbg, topdebugloc;
     if (ctx.debug_enabled) {
         // TODO: Fix when moving to new LLVM version
```

Comment:
I wonder what that TODO is about

Reply:
IIRC, it’s to put the right number in place of 0x01